examples/running-tests-using-travis-and-saucelabs/README.md

```diff
@@ -0,0 +1,9 @@
+# Example: Running Tests Using Travis CI and Sauce Labs
+
+*A set of sample files that help you learn how to run TestCafe tests in the cloud using Travis CI and Sauce Labs.*
+
+## Running the example
+
+Create a GitHub repository and copy the sample files to it.
```

Comment:
@inikulin 
Ok, I'll write about cloning the repository, but I don't yet understand why we need to run `npm install` from _examples\running-tests-using-travis-and-saucelabs_? We aren't going to run this example locally. So, it is enough to perform the following steps:
1. Clone the TestCafe repository to your machine.
git clone https://github.com/DevExpress/testcafe.git
cd examples/running-tests-using-travis-and-saucelabs
npm install

2. Create a new GitHub repository and copy the sample files from the _examples/running-tests-using-travis-and-saucelabs_ to your repository.

3. Go through steps described in the topic.

Is it right?